### ISSUE TITLE
Super jumps support as original halo 2

### ISSUE THREAD

**Issue by Andres128** [PermaNulled/cartographer#209]
It is possible to add the super jumps in halo 2, I see that it is only possible using 30fps but this configuration does not currently exist, but it would be very good because it would be very similar to the original halo 2 of the video game console xbox 1, this is very positive .

It is only a request and that is left to the user's preference if you want to play at 60fps or 30fps with super jumps, thank you very much for the excellent project and greetings.

**Comment by xxmart1naxx** [comment]
It is not about framerate but about tickrate! You can play at 60 fps with 30 Hz tickrate. Tickrate is basically the frequency at which server exchanges packets with the clients.

 However tickrate changing is not as easy as you would thing as it could cause a lot of problems. The switch from Xbox's 30Hz to PC's 60Hz already caused a lot of problems Hired Gun had to work around and they still haven't fixed everything.

 Basically just switching back to 30Hz would break all the workarounds and fixes Hired Gun have done in their effort to fix their disastrous tickrate change and basically you would also have to change all the script sleep values, timers etc. back to Xbox originals,  which would indeed take quite a lot of work.

**Comment by atticuslaurs** [comment]
This support will be in the next update -- since the OGH2 Variant will add 30 tick as a game option just put OGH2 or ogh2 in the gametype name and it will load the mod -- in the next update. Don't also use H2X though as this will over-ride the OGH2 mod and just apply H2X instead; running the game at 30 tick naturally lowers the fire-rate so it makes lowering the fire-rate through another mechanism unnecessary & it will enable super bounces ;-)
 I've been told that a match running an OGH2 gametype is fair game to super-bounce without the risk of a ban. You could I guess get banned by the server owner if they don't want you to do it on their server but you couldn't get banned from the project on a whole.

**Comment by iSupersniper** [comment]
Support added in commit: https://github.com/PermaNulled/cartographer/commit/47a9d6143fcc952a610a4d905afe90fa9e1673c7
